### ISSUE TITLE
CESMD Fetcher not finding data for Chino Hills Event

### ISSUE THREAD

**Issue by jas82** [usgs/groundmotion-processing#632]
**Describe the bug**
No data is fetched from the CESMD database for the Chino Hills Event. A search on the CESMD website fetches 422 records. Note that the CESMD website enforces a maximum record request of 200, so the data needs to be downloaded in batches.

**To Reproduce**
1. Initialize a gmrecords project: 
```gmrecords init```
2. Configure the fetcher in config/config.yml:
   ``` CESMDFetcher:
    email: [your email]@[domain]
    eq_dt: 10
    eq_radius: 10
    process_type: raw
    station_radius: 100.0
    station_type: Ground
3. Download the data: 
```gmrecords download -e ci14383980```
4. See message: 
```INFO 2021-01-26 10:59:44 | cesmd_fetcher.retrieveData: Could not find data records for M5.4 2008-07-29 18:42:15```

**Expected behavior**
The available data for this event id from the CESMD website should be fetched by gmprocess. Note that if the station_radius is reduced (say 25.0), then the data will be downloaded because the number of records is <200.

**Suggested Fix**
This could be resolved if the program constructs CESMD requests in batches of <=200 until all data is gathered and varies the station radius until the config parameter is met (near to far).

**Environment (please complete the following information):**
 - OS: macOS
 - Python Version 3.8.6
- Libcomcat Version 2.0.7
- Numpy Version 1.19.5

**Comment by emthompson-usgs** [comment]
Both, maybe? This could be resolved by CESMD removing their number of records per request limitations. But we can also fix it by updating our code to request in batches like @jas82 suggests. You do get CESMD data via our code as long as you don't request more than 200 records.

**Comment by jas82** [comment]
Maybe a warning could be output if no data is found for the event that suggests the user try narrowing the event radius. I don't see any error numbers returned by the web request that are useful for gmprocess. I think it's only 403.

**Comment by emthompson-usgs** [comment]
It looks like Mike had tried to address this when he first wrote this here: 
https://github.com/usgs/groundmotion-processing/blob/master/gmprocess/io/cosmos/cesmd_fetcher.py#L30

I'll try to debug.

**Comment by jas82** [comment]
The CESMD data folks told me that users can request up to 1,000 records if a form is filled out designating them as a bulk user and they are registered. I think the 200 limit is going to be all we have to work with so it might be best to print a message about this limit for users not familiar with the service.

**Comment by emthompson-usgs** [comment]
That's not really acceptable for our needs. I'm looking at this now, and it looks like it is actually related to the event search/matching algorithm. The way it works is really confusing though.

**Comment by emthompson-usgs** [comment]
Closed by #637.
